build-tools/biome.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
Does Biome support yaml? I would love to get a reliable formatting option for our yaml...

Reply:
No, it doesn't. We can try prettier again for that (separate from this PR) if you want. It's very likely it got better/my fears are overblown. But I'd prioritize _linting_ the YAML before formatting.

Reply:
Yeah. I was just curious.